validate-keycloak-login.py

```
@@ -0,0 +1,293 @@
+            print("   ✅ NextJS App client authentication successful")
+            print(f"   Token type: {token_data.get('token_type', 'N/A')}")
+            print(f"   Expires in: {token_data.get('expires_in', 'N/A')} seconds")
+            print(f"   Scope: {token_data.get('scope', 'N/A')}")
+            return True, token_data.get('access_token')
+        else:
+            print(f"   ❌ NextJS App client authentication failed: {response.status_code}")
+            if response.text:
+                try:
+                    error_data = response.json()
+                    print(f"   Error: {error_data.get('error_description', error_data.get('error', 'Unknown error'))}")
+                except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)